README.md

```diff
@@ -20,6 +20,10 @@ Before running `backups2datalad`, the following setup must be performed:
   version 10.20230126 is required, though you should endeavor to obtain the
   latest version.
 
+- An API token needs to be obtained for the DANDI instance that is being
+  mirrored.  When invoking `backups2datalad`, the environment variable
+  `DANDI_API_KEY` must be set to the token.
```

Comment:
do you recall if we have already any cron job on drogon which needs and uses it so we could centralize secrets specification?  couldn't recall any from a quick look.

Reply:
I guess we would need to add `source`ing the file with secrets at https://github.com/dandi/dandisets/blob/draft/tools/backups2datalad-cron-common  or do you see better place?

Reply:
I don't believe we have any other cronjobs that access the Archive, let alone with an API key.

Reply:
1ok, done changes to the script, let's see ;-)  Thank you!